content/pages/faq.md

```diff
@@ -4,7 +4,7 @@ permalink: /faq.html
 title: Frequently Asked Questions
 display_title: Frequently Asked Questions
 ---
-<main>
+<main itemscope itemtype="http://schema.org/FAQPage">
```

Comment:
around the content describing the FAQPage (after line 11 before line 16) you need : 

<div itemprop="description"> and that closed before the .feature div on line 18

Reply:
Got it—thanks for catching this! It's showing up in the tool now